Intersect.Server/Networking/PacketHandler.cs

```diff
@@ -1598,29 +1609,29 @@ public void HandlePacket(Client client, PickupItemPacket packet)
                             continue;
                         }
 
-                    var canTake = false;
-                    // Can we actually take this item?
-                    if (mapItem.Owner == Guid.Empty || Timing.Global.Milliseconds > mapItem.OwnershipTime)
-                    {
-                        // The ownership time has run out, or there's no owner!
-                        canTake = true;
-                    }
-                    else if (mapItem.Owner == player.Id)
-                    {
-                        // The current player is the owner.
-                        canTake = true;
-                    }
+                        var canTake = false;
+                        // Can we actually take this item?
+                        if (mapItem.Owner == Guid.Empty || Timing.Global.Milliseconds > mapItem.OwnershipTime)
+                        {
+                            // The ownership time has run out, or there's no owner!
+                            canTake = true;
+                        }
+                        else if (mapItem.Owner == player.Id)
+                        {
+                            // The current player is the owner.
+                            canTake = true;
+                        }
```

Comment:
I just realized there's no point in the else-if, the entire thing can be reduced down into

`var canTake = mapItem.Owner == Guid.Empty || mapItem.Owner == player.Id || Timing.Global.Milliseconds > mapItem.OwnershipTime;` but it was already an unnecessary clown fiesta so I will just make this change on my own in a separate PR.